src/main/java/com/online/cinema/dto/moviesession/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,10 @@
+package com.online.cinema.dto.moviesession;
+
+import lombok.Data;
+
+@Data
+public class MovieSessionRequestDto {
+    private Long movieId;
+    private Long cinemaHallId;
+    private String sessionTime;
```

Comment:
I think you can have a String that represents date+time here, like the `LocalDateTime` format `ISO_LOCAL_DATE_TIME` or something like it